IndexedDB/ready-state-destroyed-execution-context.html

```diff
@@ -21,11 +22,8 @@
     const openRequest = iframe.contentWindow.indexedDB.open(dbname);
     assert_equals(openRequest.readyState, 'pending');
     iframe.remove();
-    await new Promise(resolve => {
-      openRequest.onerror = resolve;
-      openRequest.onsuccess = resolve;
-    });
-    assert_equals(openRequest.readyState, 'done');
+    await load_iframe();
```

Comment:
Do we need to await _anything_ here?

The original test was authored here:

https://chromium-review.googlesource.com/c/chromium/src/+/3125334

... and the intention was just to ensure that readyState() could be called without hitting a CHECK in Chromium after the context was destroyed. 

In the PR description, you say _"make the test wait on a new iframe with the same id to be loaded, so that there is enough time for request to be completed."_ - but we don't actually care about the request being completed -- or we'd assert that it was 'done'.

So what about just removing the await entirely (added in https://phabricator.services.mozilla.com/D203270) and just keeping the assert that `readyState` returns a string (i.e. keep the next line as you suggest)

Reply:
Yes, I think the wait is unnecessary if we don't care about the value. I will update the PR.